.github/workflows/setup-shared.yml

```
@@ -0,0 +1,64 @@
+name: Shared Setup
+
+on:
+  workflow_call:
+    outputs:
+      composer-cache-hit:
+        description: "Whether composer cache was hit"
+        value: ${{ jobs.setup.outputs.composer-cache-hit }}
+      node-cache-hit:
+        description: "Whether node cache was hit"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)